pkg/agent/datapath/policyBridge.go

```diff
@@ -21,11 +21,13 @@ const (
 	EGRESS_TIER1_TABLE          = 20
 	EGRESS_TIER2_MONITOR_TABLE  = 24
 	EGRESS_TIER2_TABLE          = 25
+	EGRESS_TIER_ECP_TABLE       = 28
```

Comment:
27 maybe better, 28 & 26 remain for further usage

Reply:
maybe add a tier with monitor in future, then need 2 tables with connected serial numbers.
and tier-ecp only need to higer than tier2, so set 28